core/src/main/scala/chisel3/properties/PathType.scala

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.properties
+
+import chisel3.{Data, MemBase}
+import chisel3.experimental.BaseModule
+import firrtl.annotations.{InstanceTarget, IsMember, ModuleTarget, ReferenceTarget}
+import firrtl.ir.{PathPropertyLiteral}
+
+/** Represent a Path type for referencing a hardware instance or member in a Property[PathType]
+  */
+sealed abstract class PathType {
```

Comment:
Should this be `PathType` or perhaps just `Path`? It's `Path` in firrtl.

Reply:
renamed to `Path`